backend/routes/service.js

```diff
@@ -16,6 +17,7 @@ router.post("", extractFiles, async (req, res) => {
     priceEstimate: body.priceEstimate,
     description: body.description,
     contact: body.contactNumber,
+    creator_email: req.user,

```

Comment:
Shouldn't it be req.user.email?

Reply:
I have set the email id in req.user after decoding the token so it works